mitmproxy/dump.py

```diff
@@ -248,7 +248,7 @@ def _echo_request_line(self, flow):
         else:
             client = click.style("[replay]", fg="yellow", bold=True)
 
-        method = flow.request.method
+        method = flow.request.data.method
```

Comment:
The point of all the `strutils.bytes_to_escaped_str` calls here is that we don't want to beep on the terminal.
As we do have unicode, we should use `clean_bin` here, not `bytes_to_escaped_str`. If that still beeps (can you test that?), we should fix `clean_bin` to use the symbols from the control_pictures group for this: http://www.fileformat.info/info/unicode/block/control_pictures/images.htm

Ping me on Slack if you need help :smiley: